testing-common/src/main/java/co/elastic/otel/testing/OtelReflectionUtils.java

```diff
@@ -54,8 +54,11 @@ public static void shutdownAndResetGlobalOtel() {
         (OpenTelemetry) readField(GlobalOpenTelemetry.class, null, "globalOpenTelemetry");
     if (otel != null) {
       // unwrap from obfuscated opentelemetry
-      OpenTelemetrySdk sdk = (OpenTelemetrySdk) readField(otel, "delegate");
-      sdk.close();
+      Object sdkObject = readField(otel, "delegate");
```

Comment:
This was failing after custom exporters are added, the instance is no longer an OpenTelemetrySdk instance

Reply:
Is this still the case in the current state?
I removed the `if` check and the tests still run fine for me.

Reply:
reverted in c7b825d